deployment/ccip/changeset/cs_deploy_chain.go

```diff
@@ -119,7 +119,7 @@ func deployChainContractsForChains(
 		if !ok {
 			return fmt.Errorf("chain %d not found", chainSel)
 		}
-		if existingState.Chains[chainSel].LinkToken == nil || existingState.Chains[chainSel].Weth9 == nil {
+		if existingState.Chains[chainSel].StaticLinkToken == nil || existingState.Chains[chainSel].Weth9 == nil {
```

Comment:
This shouldn't be a replacement but an addition. If none of these three: `LinkToken`, `Weth9` and `StaticLinkToken` are available then there are no fee tokens.

I feel like either `LinkToken` or `StaticLinkToken` should always be present on all chains though.